descriptors/ModuleDescriptor-template.json

```diff
@@ -0,0 +1,124 @@
+          "modulePermissions": [
+            "inventory-storage.instances.item.get",
+            "inventory-storage.items.item.get"
+          ]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/oai/repository_info",
+          "permissionsRequired": ["oai-pmh.identify.get"],
+          "modulePermissions": []
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/oia/identifiers",
+          "permissionsRequired": ["oai-pmh.identifiers.collection.get"],
+          "modulePermissions": [
+            "inventory-storage.instances.collection.get",
+            "inventory-storage.instances.item.get"
+          ]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/oai/metadata_formats",
+          "permissionsRequired": ["oai-pmh.metadata-formats.collection.get"],
+          "modulePermissions": [
+            "inventory-storage.instances.collection.get",
+            "inventory-storage.instances.item.get",
+            "inventory-storage.items.collection.get"
+          ]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/oai/sets",
+          "permissionsRequired": ["oai-pmh.set.collection.get"],
+          "modulePermissions": []
+        }
+      ]
+    }
+  ],
+  "requires": [
+    {
+      "id": "item-storage",
+      "version": "6.0"
+    },
+    {
+      "id": "instance-storage",
+      "version": "5.0"
+    }
+  ],
+  "permissionSets": [
+    {
+      "permissionName": "oai-pmh.records.record.get",
+      "displayName": "OAI-PMH - retrieve individual record",
+      "description": "Retrieve an individual metadata record"
+    },
+    {
+      "permissionName": "oai-pmh.identify.get",
+      "displayName": "OAI-PMH - get information about a repository",
+      "description": "Get information about a repository"
+    },
+    {
+      "permissionName": "oai-pmh.identifiers.collection.get",
+      "displayName": "OAI-PMH - get headers list",
+      "description": "Get headers list of record in specific format"
+    },
+    {
+      "permissionName": "oai-pmh.metadata-formats.collection.get",
+      "displayName": "OAI-PMH - get metadata formats available from a repository",
+      "description": "Get metadata formats available from a repository"
+    },
+    {
+      "permissionName": "oai-pmh.records.collection.get",
+      "displayName": "OAI-PMH - get list of records",
+      "description": "Get records from repository"
+    },
+    {
+      "permissionName": "oai-pmh.set.collection.get",
+      "displayName": "oai-pmh - get set structure of a repository",
+      "description": "Get set structure of a repository"
+    },
+    {
+      "permissionName": "oia-pmh.all",
+      "displayName": "OAI-PMH - all permissions",
+      "description": "Entire set of permissions needed to use the inventory",
+      "subPermissions": [
+        "oai-pmh.records.record.get",
```

Comment:
again, just a reminder... if you change the permission name, don't forget to update it here too